### DISCUSSION TITLE
Use acode X to connect to remote server (linux)

### DISCUSSION THREAD

**Discussion by vikesh001** [bajrangCoder/acode-plugin-acodex#81] [Ideas]
I’d like to propose an idea to further improve the functionality of Acode X, especially for developers who rely on it for remote server access and development workflows. While Acode X is already a powerful tool for connecting to remote servers via SSH or VNC, executing commands, and running GUI apps, there’s potential to expand its capabilities even further.

**Comment by bajrangCoder** [comment]
Remote connection through ssh is already in todo and soon it will be available and that will work without starting server etc

**Reply by vikesh001** [reply]
thank you for your for reply
this feature helps me a lot

**Comment by jstiggles83-collab** [comment]
Fuck all you mutherfuckers. What you are doing is internet crimes. stealing websites, companies, and people's livelihood. I'm hot on your trail. I would be very worried if I were you.
